lib/dvb/epgcache.cpp

```diff
@@ -1607,8 +1609,9 @@ int handleEvent(eServiceEvent *ptr, ePyObject dest_list, const char* argstring,
 				Py_DECREF(nowTime);
 			Py_DECREF(convertFuncArgs);
 			Py_DECREF(dest_list);
-			PyErr_SetString(PyExc_Exception, "[eEPGCache] handleEvent: error in convertFunc execute");
-			//eDebug("[eEPGCache] handleEvent: error in convertFunc execute");
+			PyErr_SetString(PyExc_StandardError,
```

Comment:
PyExc_StandardError is wrong!
And you don't need eDebug if you have a python exception.